### ISSUE TITLE
GeoServer: deactivate default data 

### ISSUE THREAD

**Issue by JakobMiksch** [klips-project/klips-sdi#56]
On startup the GeoServer docker image creates default data stores and layers. We should deactivate this behaviour.

**Comment by weskamm** [comment]
does this also happen when you start the image with an empty mounted geoserver_data image?

**Comment by JakobMiksch** [comment]
yes the demo data appears after starting the container on a clean system

**Comment by weskamm** [comment]
an empty data dir should have circumvent the original data, hmm

**Comment by JakobMiksch** [comment]
I tried it with have no `geoserver_data` directory at the beginning and also having an empty `geoserver_data` directory.
In both cases the default data is present.

**Comment by weskamm** [comment]
ok thats good to know, did not expect it this way

**Comment by JakobMiksch** [comment]
there is some progress https://github.com/geoserver/docker/pull/9
we wait until the decision and then adapt

**Comment by buehner** [comment]
FYI I just merged https://github.com/geoserver/docker/pull/9
